heartbeat/galera

```diff
@@ -488,6 +501,65 @@ detect_first_master()
     set_bootstrap_node $best_node
 }
 
+galera_status()
+{
+    local loglevel=$1
+    local rc
+
+    if [ -e $OCF_RESKEY_pid ]; then
+        mysql_common_status $loglevel
+        rc=$?
+    else
+        # if pidfile is not created, the server may
+        # still be starting up, e.g. running SST
+        ps auxww | grep -v -e "${OCF_RESKEY_binary}" -e grep | grep -qe "--pid-file=$OCF_RESKEY_pid"
```

Comment:
The narrowing to the explicit "pidfile" is a fairly good guarantee that I'm monitoring the mysqld process I'm interested in (because the RA won't start a mysqld if the pidfile existed beforehand).
If we only monitor mysqld, we makes the start process more fragile: we may miss the original pid failure (e.g if two different mysql are legitimately running on the same machine) and keep the node in Slave state forever.

Note that the handy workaround of "copying pidfile" still work in the same extent as before:
  . one can still copy the pidfile while pacemaker resource is unmanaged, then cleanup, then manage. The monitor op will detect galera as running, as before.
  . in the old agent, if a slave node didn't finish sync, there is no pidfile to copy. So the strict grep doesn't restrain the use case. 

Last point, since the boot process now depends on a attribute "sync-needed" in the CIB, making the "copy pidfile" workaround works also during the sync would require additional manual steps. So weakining the grep for that didn't seem justified IMHO.